src/components/BodyWithResults/BodyWithResults.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
     movies: Movie[];
     handleMovieCardClick: (title: string) => void;
     genre: string;
+    isEmpty: boolean;
 }
 
 export default React.memo((props: Props) => {
```

Comment:
How about using props destructing? `{ movies, genre, ... }: Props` instead of `props: Props`

Reply:
I don't like the idea to destruct props and state into variables, because it will be hard for me to understand what is this variable from, during code review without the full file visibility

Reply:
Still, this is a common way to write functional components.